internal/core/domain/market.go

```diff
@@ -11,9 +12,9 @@ import (
 	"github.com/tdex-network/tdex-daemon/pkg/mathutil"
 )
 
-type FixedFee struct {
-	BaseFee  uint64
-	QuoteFee uint64
+type MarketFee struct {
+	BaseAsset  uint64
```

Comment:
Should we rename BaseAsset to BaseAssetFee? same for QuoteAsset

Reply:
it seems to me like a repetition based on the name given to the Market fields:
i think `market.PercentageFee.BaseAsset` is better than `market.PercentageFee.BaseAssetFee`.